src/components/App/index.tsx

```diff
@@ -2,6 +2,7 @@ import { Routes, Route } from "react-router-dom";
 import Home from "../Home";
 import GoLive from "../GoLive";
 import Login from "../Login";
+import Login2 from "../Login2";
```

Comment:
change name to something better, not Login2

Reply:
'Login' and 'Login2' renamed to 'SignIn' and 'SignInDetails'